packages/wonder-blocks-form/src/components/radio-core.tsx

```diff
@@ -60,7 +60,7 @@ const disabledChecked = {
     height: size / 2,
     width: size / 2,
     borderRadius: border.radius.radius_full,
-    backgroundColor: semanticColor.action.primary.disabled.background,
+    backgroundColor: semanticColor.core.border.disabled.strong,
```

Comment:
Nice! I'm glad we're adding the core semantic colors. It makes a lot of sense to me to use the core tokens instead of the action tokens for things like forms!